internal/proto/controller/api/resources/targets/v1/target.proto

```diff
@@ -297,7 +297,7 @@ message SshTargetAttributes {
       this: "attributes.enable_session_recording"
```

Comment:
Should we not include the storage bucket id?

Reply:
No, these new features are going to be for phase 2 of the telemetry

Reply:
Why is enable_session_recording set then?